packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js

```diff
@@ -1,8 +1,7 @@
 import cuid from 'cuid'
 import PropTypes from 'prop-types'
-import React, { useEffect } from 'react'
+import React, { useEffect, useRef, useState } from 'react'
```

Comment:
[minor] I'm not sure where this branch is in regards to its base, but we've got an `import React` here that'll be reintroduced(??) after that standard was removed recently in #2288

Reply:
I ran the React import codemod across the NextJS apps but I've left the libraries alone. This shouldn't make a difference at the moment.